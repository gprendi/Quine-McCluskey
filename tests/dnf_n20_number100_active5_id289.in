d 20
00000000000110000000
00000000000000010000
01000000000001000000
00000000000000000000
00000000000100010000
01000000000100010000
00000000000111000000
01000000000110010000
00000000000101010000
00000000000010010000
01000000000111000000
00000000000011000000
01000000000101000000
00000000000101000000
01000000000001010000
01000000000100000000
00000000000001010000
01000000000111010000
00000000000110010000
01000000000000000000
00000000000001000000
01000000000110000000
00000000000111010000
00000000000011010000
01000000000011010000
01000000000011000000
00000000000100000000
01000000000010010000
01000000000000010000
