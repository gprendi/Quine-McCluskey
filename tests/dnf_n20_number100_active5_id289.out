p dnf 20 0
-1 -2 -3 -4 -5 -6 -7 -8 -9 -10 -11 12 -15 -17 -18 -19 -20 0
-1 -2 -3 -4 -5 -6 -7 -8 -9 -10 -11 -13 -15 -17 -18 -19 -20 0
-1 -2 -3 -4 -5 -6 -7 -8 -9 -10 -11 14 -15 -17 -18 -19 -20 0
-1 -2 -3 -4 -5 -6 -7 -8 -9 -10 -11 -15 16 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 -12 -13 -15 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 -12 14 -15 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 -12 -15 16 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 12 13 -15 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 12 -14 -15 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 12 -15 -16 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 -13 -14 -15 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 -13 -15 -16 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 13 14 -15 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 13 -15 16 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 -14 -15 16 -17 -18 -19 -20 0
-1 -3 -4 -5 -6 -7 -8 -9 -10 -11 14 -15 -16 -17 -18 -19 -20 0
